src/modules/dashboard/components/UnlockTokenDialog/UnlockTokenForm.tsx

```diff
@@ -81,7 +88,11 @@ const UnlockTokenForm = ({
         </DialogSection>
       )}
       <DialogSection appearance={{ theme: 'sidePadding' }}>
-        <FormattedMessage {...MSG.description} />
+        {colony.isNativeTokenLocked ? (
+          <FormattedMessage {...MSG.description} />
+        ) : (
+          <FormattedMessage {...MSG.unlockedDescription} />
+        )}
       </DialogSection>
       <DialogSection appearance={{ theme: 'sidePadding' }}>
```

Comment:
Yeah makes sense was silly to leave it, will hide the notice when unlocked. Thanks Raul

Reply:
Fixed now on last commit, I added an unlocked style to add margin-bottom 20px, since hiding the notice made the description have no margin-bottom.

![image](https://user-images.githubusercontent.com/6601142/144904976-183203bf-e8b2-41cb-98af-fe42da0de3a0.png)